rootex/core/renderer/shaders/decal_vertex_shader.hlsl

```diff
@@ -0,0 +1,56 @@
+#include "register_locations_vertex_shader.h"
```

Comment:
do we really need a custom vertex shader for decals now?

Reply:
Removed it.